core/templates/pages/topic-editor-page/editor-tab/topic-editor-tab.directive.spec.ts

```diff
@@ -648,4 +652,49 @@ describe('Topic editor tab directive', function() {
     topicReinitializedEventEmitter.emit();
     expect(ctrl.initEditor).toHaveBeenCalledTimes(2);
   });
+
+  it('should call the TopicUpdateService if skillId is added in the ' +
+     'diagnostic test', fakeAsync(function() {
+    var updateSkillIdForDiagosticTestSpy = spyOn(
+      TopicUpdateService, 'updateDiagnosticTestSkills');
+    $scope.selectedSkillForDiagnosticTest = skillSummary;
+    $scope.availableSkillSummariesForDiagnosticTest = [skillSummary];
+    $scope.addSkillForDiagnosticTest();
+    $rootScope.$apply();
+    tick();
+    expect(updateSkillIdForDiagosticTestSpy).toHaveBeenCalled();
```

Comment:
Maybe also verify that the arguments are the right ones. Ditto below/elsewhere.

Reply:
Added the arguments to verify here and below.